README.md

```diff
@@ -21,6 +21,12 @@ cargo audit bin target/release/your-project
 
 `cargo auditable` works with any Cargo command. All arguments are passed to `cargo` as-is.
 
+## Adoption
+
+Microsoft uses `cargo auditable` internally and maintains the [data extraction library for Go](https://github.com/microsoft/go-rustaudit).
+
+[NixOS](https://nixos.org/) and [Void Linux](https://voidlinux.org/) build all their Rust packages with `cargo auditable`. If you install packages from Nix, you can audit them!
```

Comment:
@Shnatsel I've switched from Void Linux to Chimera Linux, and have contributed similar changes to Chimera Linux as well, see https://github.com/chimera-linux/cports/pull/1598